cluster/pulumi/common-validator/src/config.ts

```diff
@@ -6,7 +6,7 @@ import {
   KmsConfigSchema,
   LogLevelSchema,
 } from '@lfdecentralizedtrust/splice-pulumi-common/src/config';
-import { clusterSubConfig } from '@lfdecentralizedtrust/splice-pulumi-common/src/config/configLoader';
+import { clusterSubConfig } from '@lfdecentralizedtrust/splice-pulumi-common/src/config/config';
```

Comment:
let's make sure the pulumi automation calls still work, changing imports especially around the configs has a tendency to fully break the automation by importing the pulumi runtime